packages/stylelint-plugin/src/stylelintPlugin.spec.ts

```diff
@@ -1,37 +1,14 @@
-import { Root } from "postcss"
-import { Language, Options } from "./types"
-import { allUsedTokensMustBeImported } from "./rules/all-used-tokens-must-be-imported"
-import { allTokenImportsMustBeUsed } from "./rules/all-token-imports-must-be-used"
-import { preferVarTokens } from "./rules/prefer-var-tokens"
-import { noInvalidUseOfVarTokensInEquations } from "./rules/no-invalid-use-of-var-tokens-in-equations"
-import { noInvalidUseOfVarTokensInFunctions } from "./rules/no-invalid-use-of-var-tokens-in-functions"
-import { noTokensInVariables } from "./rules/no-tokens-in-variables"
-import { getParser } from "./util/utils"
+import { lint } from "stylelint"
```

Comment:
The changes in this file might be pretty hard to comprehend.
To summarise:
 - Rename variables within expected outputs so that they match our new design-tokens
 - Rename imports too (because we're moving from color-vars to color etc)
 - In some cases, input source was actually invalid, for example non existent variables we're used. These needed to change because improvements we're made to the rules.

Reply:
Oh and also, the number of expected warnings sometimes changed too.

Reply:
Another important bit about this file: Once we break design-tokens, these tests will break too.
Reason being that the stylelint plugin will behave differently for tokens that don't exist within design-tokens (for example, it won't recommend importing scss modules that don't exist, or it will remove scss imports for old variables like `kz-color-wisteria-100`)